chat_stream.go

```diff
@@ -9,6 +9,7 @@ type ChatCompletionStreamChoiceDelta struct {
 	Content      string        `json:"content,omitempty"`
 	Role         string        `json:"role,omitempty"`
 	FunctionCall *FunctionCall `json:"function_call,omitempty"`
+	ToolCalls    []ToolCall    `json:"tool_calls,omitempty"`
```

Comment:
I don't see this field in the docs, is it undocumented yet? https://platform.openai.com/docs/api-reference/chat/streaming

Reply:
Yes documented here: "The chat completion chunk object" -> choices -> delta -> tool_calls

Reply:
Thanks! Also noted: lots of things are marked as deprecated now, we should probably reflect those too https://github.com/sashabaranov/go-openai/issues/540